src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SkippableFactAttribute.cs

```diff
@@ -12,13 +13,13 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 /// </summary>
 internal class SkippableFactAttribute : FactAttribute
 {
-    public SkippableFactAttribute(string envName, bool skipOnNullOrWhiteSpace = false, bool skipOnTrue = false) =>
-        CheckEnvs(skipOnNullOrWhiteSpace, skipOnTrue, (skip) => Skip = skip, envName);
+    public SkippableFactAttribute(string envName, bool skipOnNullOrWhiteSpace = false, bool skipOnTrue = false, string[] skipArchitectures = null) =>
```

Comment:
With the introduction of skipArchitectures, the skipOnNullOrWhiteSpace and skipOnTrue parameter names can be confusing.  They are tied to the envName but skipArchitectures isn't.  Renaming skipOnNullOrWhiteSpace and skipOnTrue in such a way to associate them to the envName would be helpful.

```suggestion
    public SkippableFactAttribute(string envName, bool skipOnNullOrWhiteSpaceEnv = false, bool skipOnTrueEnv = false, string[] skipArchitectures = null) =>
```